index.html

```diff
@@ -1627,7 +1638,7 @@ <h2>Affordances</h2>
             <p>
                 A central aspect in W3C WoT is the provision of machine-understandable metadata (i.e., <a>WoT Thing Descriptions</a>).
                 Ideally, such metadata is self-descriptive, so that <a>Consumers</a> are able to identify
-                <em>what</em> capabilities a <a>Thing</a> provides and <em>how</em> to use the provided capabilities.
+                <em>which</em> capabilities a <a>Thing</a> provides and <em>how</em> to use the provided capabilities.
```

Comment:
Please leave "what". "Which" has the notion of a well-known list from which must be picked.

Reply:
ok